matrix-sdk/src/main/java/org/matrix/androidsdk/data/RoomAccountData.java

```diff
@@ -54,8 +54,8 @@
     // The events the user wants to hide in this room.
     private Map<String, TaggedEventInfo> hiddenEvents = null;
 
-    // Store the content of all the provided events by using their event type.
-    private Map<String, JsonObject> eventContentsMap = new HashMap<>();
+    // Store all the provided events by using their event type.
+    private Map<String, Event> eventsMap = new HashMap<>();
```

Comment:
Does an upgrade of the application with this change will work? I mean when the SDK will deserialize RoomAccountData, maybe this will cause issue.

Reply:
I consider the problem is already here. Currently the deserialisation failed on RoomAccountData, which is the bug I want to fix here. But you're right I will test again an update before merging

Reply:
ok